From: Adrian Perez de Castro <aperez@igalia.com>
Date: Mon, 11 May 2020 10:07:41 +0300
Subject: Fix build issue due to missing VK_RESULT_RANGE_SIZE

Conditionally check whether the Vulkan headers version defines
VK_RESULT_RANGE_SIZE, and avoid using it for version >=140. The
following comming in Vulkan-Headers has removed the enum value:

  https://github.com/KhronosGroup/Vulkan-Headers/commit/0c5351f5e9114d3e9033aeae51d036a3d201c082#diff-4febd94c0666d59030d8b1dd20c72403

(cherry picked from commit 5c4ca1424d7d90a92f15579fe9d14fbb9952d7d7)
---
 gdk/gdkvulkancontext.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/gdk/gdkvulkancontext.c b/gdk/gdkvulkancontext.c
index 7112e67..be249fc 100644
--- a/gdk/gdkvulkancontext.c
+++ b/gdk/gdkvulkancontext.c
@@ -223,7 +223,9 @@ gdk_vulkan_strerror (VkResult result)
     case VK_ERROR_PIPELINE_COMPILE_REQUIRED_EXT:
       return "A requested pipeline creation would have required compilation, but the application requested compilation to not be performed.";
 #endif
+#if VK_HEADER_VERSION < 140
     case VK_RESULT_RANGE_SIZE:
+#endif
     case VK_RESULT_MAX_ENUM:
     default:
       return "Unknown Vulkan error.";
